tools/tick-cluster.js

```diff
@@ -570,6 +630,7 @@ function displayMenu(logFn) {
     logFn('\tK\t\tRevive suspended or killed processes');
     logFn('\tl <count>\tSuspend processes');
     logFn('\tm <count>\tTerminate processes');
+    logFn('\tr <batch size>\tRestart processes');
```

Comment:
I think the list is in alphabetic order, except for this line. Lets move it down a bit.